src/bp/core/repositories/sessions.ts

```diff
@@ -136,7 +136,7 @@ export class KnexSessionRepository implements SessionRepository {
   async update(session: DialogSession, trx?: Knex.Transaction): Promise<void> {
     const req = this.database
       .knex(this.tableName)
-      .where('id', session.id)
+      .where({ id: session.id, botId: session.botId })
```

Comment:
@allardy & @laurentlp  is there a reason why sessionId doesn't contain botId anyway ? it would be easier to simply have it in the session instead of always have to use `.where({id, botId})`

Reply:
I agree! It might break the current active sessions, but that's pretty much the only side effect I can see.